assets/js/video-modal.js

```diff
@@ -5,28 +5,34 @@ var modal = document.getElementById("modal");
 var btn = document.getElementById("video-modal");
 
 // Get the video control
-var video = document.getElementById("video");
+var vid = document.getElementById("video");
 
-// Get the <span> element that closes the modal
-var span = document.getElementsByClassName("close-video")[0];
+// Get the <h3> element that closes the modal
+var cross = document.getElementsByClassName("close-video")[0];
```

Comment:
I guess since we are targeting the `h3` element with the `cross` sign, the class name should be `video` instead of `close-video` and since there are two elements with video as class and our target is the second one, so this line of code should be
`var cross = document.getElementsByClassName("video")[1]`
with this we will be targeting the correct cross button